cpp/src/rolling/rolling.cu

```diff
@@ -44,6 +44,7 @@ std::unique_ptr<column> rolling_window(column_view const& input,
                "Defaults column must be either empty or have as many rows as the input column.");
 
   if (agg.kind == aggregation::CUDA || agg.kind == aggregation::PTX) {
+    // TODO: In future, might need to clamp preceding/following to column boundaries.
```

Comment:
So why not now?

Reply:
Modifying UDF behavior is out of scope for this change. 

I would need to either materialize the offsets, or make the UDF code work with a generic iterator.